apps/ui/lib/lens/misc/Inbox/Inbox.tsx

```diff
@@ -122,14 +122,27 @@ const Inbox = ({ channel }: Props) => {
 		<Column pt={2}>
 			<Tabs>
 				<Tab title="Open">
-					<InboxTab query={openQuery} channel={channel} canArchive={true} />
+					<InboxTab
+						query={openQuery}
+						channel={channel}
+						canArchive={true}
+						isDirectPing={false}
+					/>
 				</Tab>
 				<Tab title="Archived">
 					<InboxTab query={archivedQuery} channel={channel} />
 				</Tab>
 				<Tab title="Sent">
 					<InboxTab query={sentQuery} channel={channel} />
 				</Tab>
+				<Tab title="Direct Pings">
```

Comment:
So we're going with a seperate tab? Not a problem, but did you check in with @JonJRich and/or @Jazzagi just in case?

Reply:
@joshbwlng They both said new tab is good